wellregistry/wellregistry/templates/admin/bulk_upload.html

```diff
@@ -2,42 +2,55 @@
 {% load static %}
 
 {% block breadcrumbs %}
-    <div class="breadcrumbs">
-      <a href="{% url 'admin:index' %}">Home</a>
-        &rsaquo; <a href="{% url 'admin:registry_monitoringlocation_changelist' %}">Monitoring Location</a>
-        &rsaquo; Bulk Upload
-    </div>
+<div class="breadcrumbs">
+    <a href="{% url 'admin:index' %}">Home</a>
+    &rsaquo; <a href="{% url 'admin:registry_monitoringlocation_changelist' %}">Monitoring Location</a>
+    &rsaquo; Bulk Upload
+</div>
```

Comment:
I wonder what happened to these breadcrumbs?

All this whitespace change makes it hard to see the new changes.

Reply:
white spaces fixed.. Breadcrumbs showing for me

Reply:
spacing fixed